client/lib/Navbar.html

```diff
@@ -0,0 +1,78 @@
+<nav>
+	<button class="hamburger open" on:click="set({visible: true})"></button>
+</nav>
+
+{{#if visible}}
+<div class="anythingelsebutton" on:click="set({visible: false})"></div>
+{{/if}}
+
+<div class="slide-out {{visible}}Visible">

```

Comment:
I'd strongly recommend using data attributes instead of dynamic classes to represent state:

```html
<div data-visible="{{visible}}" class="slide-out">
```

Then you can use the selectors `[data-visible=true]` and `[data-visible=false]`.